examples/tests/valid/util.env.w

```diff
@@ -0,0 +1,26 @@
+bring cloud;
+bring util;
+
+let random_name = "RANDOM123412121212kjhkjskdjkj";
+
+assert(util.Util.env("PATH").length > 0);
+
+let var failed = false;
+try {
+  util.Util.env(random_name);
+} catch {
+  failed = true;
+}
+assert(failed);
+
+// TODO: this is not supported yet (P1)
+/*
```

Comment:
@yoav-steinberg is this supposed to be resolved by https://github.com/winglang/wing/pull/2158?